src/PIL/GimpPaletteFile.py

```diff
@@ -24,33 +25,49 @@ class GimpPaletteFile:
 
     rawmode = "RGB"
 
-    def __init__(self, fp):
+    #: override if reading larger palettes is needed
+    max_colors = 256
+    _max_line_size = 100
```

Comment:
We're reading RGB data from each line. That's three integers. Under what circumstances would a user need to read more than 100 characters?

Reply:
The 4th Colunn contains the color name, though that is currently unused. And there might be a name with more than 100 characters - moving the number to an attribute just offera chance of one working around an error imposed by an arbitrary, though reasonable, limit on an otherwise good file.

Reply:
(currently unused in PIL, that is: GIMP uses that name)